src/test/resources/com/znsio/teswiz/features/jioCinema.feature

```diff
@@ -9,4 +9,13 @@ Feature: jio cinema test
     When I swipe left on tending in india section
     Then I am able to view number 3 trending movie
     When I swipe right on tending in india section
-    Then I am able to view number 1 trending movie
\ No newline at end of file
+    Then I am able to view number 1 trending movie
+
+ #  CONFIG=./configs/jiocinema_local_config.properties PLATFORM=android TAG=horizontalSwipeUsingGesturePluginTest ./gradlew run
+  @android @swipe @horizontalSwipeUsingGesturePluginTest
+  Scenario: validating swipe functionality
+    Given I navigate to jio Cinema application's home page
+    When I swipe "left" trending no 2 on trending in india section
+    Then I am able to view number 3 trending movie
```

Comment:
I think instead of writing 3 trending movies - you should write
I am able to view number  trending movie - the movie count. may change later on

Reply:
number 3 is not the count of movie, that is the trending position, which changes everytime.